consumer.go

```diff
@@ -89,7 +89,7 @@ func AcquireConsumer(db *sql.DB, cfg *config.Config, resumptionTime int64, useBl
 	if cfg.LightSeed != 0 {
 		lastNumber = cfg.LightSeed
 		log.Info("flume light service initiated from lightSeed value", "block", cfg.LightSeed)
-		consumer, err := deliverConsumer(brokerParams, resumption, reorgThreshold, resumptionTime, lastNumber, lastHash, lastWeight, append(trackedPrefixes, ptp...))
+		consumer, err := deliverConsumer(brokerParams, resumption, reorgThreshold, resumptionTime, lastNumber, lastHash, lastWeight, append(trackedPrefixes, ptp...), cfg.WhitelistExternal)
```

Comment:
This seems like not a part of this PR

Reply:
Its pork. I should have called that out.